### PR TITLE
OSSM-6640 OSSM Release Notes 2.5.z Restructure

### PR DESCRIPTION
**Pull request by gwynnemonahan**
[OSSM-6640](https://issues.redhat.com//browse/OSSM-6640) OSSM Release Notes 2.5.z Restructure

Restructure rel notes for 2.6.0, 2.5.3, 2.4.9; 2.5.2, 2.4.8, and 2.3.12.

ContentX Initiative.

All content has been previously reviewed/approved/merged in prior releases. This PR is restructuring already existing content to make it more user friendly for the End User, and the Red Hat Writer responsible for compiling and producing Red Hat Service Mesh Release Notes.

It is a ContentX initiative. 

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSSM-6640

Link to docs preview:

https://76920--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ossm-release-2-3-12.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+|Kiali
+|1.65.z
+|===
+
+//[id="ossm-fixed-issues-2-3-12"]
+//=== Fixed issues
+
+
+//[id="ossm-known-issues-2-3-12"]
+//=== Known issues
```
Comment (abrennan89) [inline modules/ossm-release-2-3-12.adoc:42]: I think these should be H2 headings (`==`) instead of `===`

File: modules/ossm-release-2-3-12.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-version-2-3-12_{context}"]
+== {SMProductName} version 2.3.12
```
Comment (abrennan89) [inline modules/ossm-release-2-3-12.adoc:8]:
```suggestion
[id="ossm-release-2-3-12_{context}"]
= {SMProductName} version 2.3.12
```

For all the modules, ID names should be the same as the filenames, and the first heading in the file should be a H1 (`=`)

**Comment by gwynnemonahan** [comment]
/label peer-review-needed

Feel free to DM me with questions, or thoughts/suggestions. There are some challenges with OSSM Release Notes that may not be readily apparent.

File: modules/ossm-release-2-4-8.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+//Includes 2.5.2, 2.4.8, 2.3.12
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.12 and later.
+
+The most current version of the {SMProductName} Operator can be used with all supported versions of {SMProductShortName}. The version of {SMProductShortName} is specified using the `ServiceMeshControlPlane`.
+
+[id="ossm-component-updates-2-4-8_{context}"]
+== Component updates
+
+// Release is scheduled for May 22, 2024. Code and Doc Freeze is scheduled for May 10, 2024. Component versions should be available after May 10.
```
Comment (skrthomas) [inline modules/ossm-release-2-4-8.adoc:21]: Can this old note be removed or updated?

**Review by skrthomas — COMMENTED** [review]
The restructure looks good to me. Just a few comments but overall organization is OK. Tricky that the 2.6, 2.5.3, and 2.4.9 are one release, and that 2.5.2 is a separate release but I think you've handled that best you can given the sorta, seemingly-out-of-order release versioning

File: modules/ossm-release-2-5-2.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+////
+
+This release of {SMProductName} includes versions 2.4.8 and 2.3.12, addresses Common Vulnerabilities and Exposures (CVEs), and contains bug fixes.
+
+This release is supported on {product-title} 4.12 and later.
+
+[id="ossm-component-updates-2-5-2-only_{context}"]
+== Component updates
+
+// Release is scheduled for May 22, 2024. Code and Doc Freeze is scheduled for May 10, 2024. Component versions should be available after May 10.
```
[the remark this answers is not in the document]
Comment (gwynnemonahan) [reply modules/ossm-release-2-5-2.adoc:23]: It's for historical purposes for management.

File: modules/ossm-release-2-4-8.adoc (hunk 3 above)
Comment (gwynnemonahan) [reply modules/ossm-release-2-4-8.adoc:21]: It's for historical purposes for management.

**Comment by gwynnemonahan** [comment]
Rebased.

No longer cherry-picking OSSM 2.x back to 4.12 or 4.13, so did not rebase against 4.12 or 4.13.

**Pull request merged**